test/unittests/iotools/converters/genfi_to_bids/test_genfi_to_bids_utils.py

```diff
@@ -11,8 +20,129 @@
         ("T2", "T2w"),
         ("fieldmap", "fieldmap"),
         ("fmri", "rsfmri"),
-        ("blzflbzv", None),
     ],
 )
 def test_identify_modality(input, expected):
     assert identify_modality(input) == expected
+
+
+def test_identify_modality_is_nan():
+    import math
```

Comment:
Maybe put the import at the top to be consistent with what you did for the other test functions.